SourceryRuntime/Sources/AST/Type.swift

```diff
@@ -255,6 +274,9 @@ public typealias AttributeList = [String: [Attribute]]
     public var based = [String: String]()
 
     // sourcery: skipEquality, skipDescription
+    /// Types this type inherits from or implements, including unknown (not scanned) types with extensions defined
+    public var basedWithTypes = [String: Type]()
```

Comment:
Not sure that's the best name, just `basedTypes`?

Reply:
changed, thanks